source/layout/theme.liquid

```diff
@@ -10,6 +10,11 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
+  <script>
+    // remove no-js and add 'js' to the HTML
```

Comment:
Yes I know. But currently there is no use for it :)
I'll still keep your changes because it might be useful for images which are lazy loaded.